Userland/Libraries/LibWeb/Painting/ImagePaintable.cpp

```diff
@@ -53,8 +53,8 @@ void ImagePaintable::paint(PaintContext& context, PaintPhase phase) const
 
     if (phase == PaintPhase::Foreground) {
         auto image_rect = context.rounded_device_rect(absolute_rect());
+        auto& image_element = verify_cast<HTML::HTMLImageElement>(*dom_node());
```

Comment:
`ImagePaintable` can also have a `HTMLObjectElement` DOM node, which would break this cast.

Reply:
How do I create such an Element in HTML?

I've changed the cast to a DOM::Element, that should work for both cases.